dogstatsd.py

```diff
@@ -578,15 +694,34 @@ def main(config_path=None):
             return 1
 
         if command == 'start':
-            daemon.start()
+            if dsd6:
+                dsd6.execute([dsd6_path, command], env)
```

Comment:
I don't think we should support these commands with dogstatsd6, except for the `info` one.

Dogstatsd with dogstatsd6 enabled should either be run in the foreground (by supervisor generally), i.e. with no `command`, or with the `info` command.